src/utils/network-monitor.ts

```diff
@@ -1016,11 +1016,10 @@ export class NetworkMonitor {
       canFail: true,
     })
     if (block !== undefined && block !== null && 'transactions' in block) {
-      const recentBlock: boolean = this.currentBlockHeight[job.network] - job.block < 5
+      const recentBlock = true // this.currentBlockHeight[job.network] - job.block < 5
```

Comment:
what's this change about

Reply:
Temporary change to get gas pricing from current blocks (even if out of sync), while I finish up the background constant gas pricing watcher.